README.md

```diff
@@ -35,11 +35,27 @@ The Container Gateway plugin requires a Pulp 3 instance to connect to.  Related
 
 # Database information
 
-SQLite database migrations are completely automated.  The plugin checks if the database is up-to-date before each query.
+SQLite and PostgreSQL are supported, with SQLite being the default for development and testing.
+Use PostgreSQL in production for improved performance by adding the following settings:
+```
+# Example PostgreSQL connection settings (default port is 5432)
+:database_backend: postgresql
+:postgresql_host: localhost
+:postgresql_port: 5432
+:postgresql_user: foreman-proxy
+:postgresql_database: container_gateway
+:postgresql_password: changeme
```

Comment:
Thoughts around exposing the DB URL as a parameter instead of DB specific parameters? I think it would greatly simplify things. Fewer options, less code to pass around.